pageserver/src/basebackup.rs

```diff
@@ -193,6 +192,17 @@ impl<'a> Basebackup<'a> {
             let header = new_tar_header(&path, buf.len() as u64)?;
             self.ar.append(&header, &buf[..])?;
         }
+        else
+        {
+            // If the transaction is not in progress anymore,
```

Comment:
This should never happen AFAICS. I believe we can remove `get_tx_is_in_progress` altogether.